### ISSUE TITLE
PSX games crash (Beetle HW and PCSX ReARMed)

### ISSUE THREAD

**Issue by tytydraco** [Swordfish90/LibretroDroid#50]
I've replicated this using the Sample Activity found within your project. Loading either of the PSX cores (even with a valid rom) causes this error:

`A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 20817 (GLThread 33542), pid 20778 (.android.gl2jni)`

You can try to replicate the error and see if it is just my device, since my device also has the weird skipping issue (present in the Sample Activity as well when configChanges are removed).

I tested this exact core in RetroArch and it works fine.

**Comment by Swordfish90** [comment]
Hi @tytydraco . I just gave this a go and I'm not able to reproduce the issue. I tried both with pcsx_rearmed bundled within Lemuroid and the latest build from buildbot.

I also took the liberty to look inside the Ludere source code and I think the issue is the way you load games. It looks like you're loading every ROM as bytes. Some core requires a file path (if the game is too big to fit memory or needs auxiliary files such as bin/cue), but the loadGameFromBytes passes nullptr, and that's likely the null reference you're seeing afterward.
